presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/SpatialPartitioningInternalAggregateFunction.java

```diff
@@ -88,8 +90,7 @@ public static void combine(SpatialPartitioningState state, SpatialPartitioningSt
     public static void output(SpatialPartitioningState state, BlockBuilder out)
     {
         if (state.getCount() == 0) {
-            out.appendNull();
-            return;
+            throw new PrestoException(INVALID_SPATIAL_PARTITIONING, "No rows supplied to spatial partition.");
```

Comment:
@jagill I just realized that this error code means that provided spatial partitioning is not valid. Hence, it cannot be used here. Could you use something else? For example, INVALID_FUNCTION_ARGUMENT?